static/js/jira-select-card-option.js

```diff
@@ -0,0 +1,8 @@
+$(".optionsCard").click(function (event) {
+	event.preventDefault();
+	const selectedVersion = $(event.target).data("type");
```

Comment:
Okay this is really minor. I am happy with the code as it it. It is just suggestions:

It seems we don't need the `.data("type")` here. We can just do the following:

```javascript
$(".optionsCard").click(function (event) {
	event.preventDefault();
	$(".optionsCard").removeClass("selected");
  $(event.target).closet('.optionsCard').addClass('selected');
	$(".optionBtn").prop("disabled", false).attr("aria-disabled", "false").addClass("aui-button-primary");
});
```

So @mboudreau opinions? You mentioned the "data('type')" yesterday, I want to ask your thoughts. 

---

for [this](https://stackoverflow.com/a/10086501), we could replace the  
`$(event.target).closet('.optionsCard').addClass('selected');`
to
`$(event.currentTarget).addClass('selected');`